src/Model.hs

```diff
@@ -12,7 +12,7 @@ data ImageRequest = ImageRequest {
   image :: String
 } deriving (Show, Generic)
 
-data ColorsResponse = ColorsResponse { 
+newtype ColorsResponse = ColorsResponse { 
```

Comment:
co tu sie zmienilo ze tak?

Reply:
HLint mówi że jak jest jeden konstruktor to żeby dawać newtype. Ja mu ufam

Reply:
aaa to moze byc bardziej optymalne, spoczko

Reply:
newtype pod spodem sie rozwiazanie do tego jednego atrybutu